src/vanilla.ts

```diff
@@ -97,7 +97,7 @@ export const classesArray = (args:string[]|string[][]):string[] => {
 	var classes:string[] = [];
 	iterate( args, (_classes) =>{
 		if( typeof _classes === 'string' ){
-			_classes = _classes.trim().split(/[\11\12\14\15\40]/);
+			_classes = _classes.trim().split(/[\x09\x0A\x0C\x0D\x20]/);
```

Comment:
I like the approach taken in #748, it makes it more immediately obvious what characters these are:

```suggestion
			_classes = _classes.trim().split(/[\t\n\f\r\s]/);
```